jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -23,8 +23,33 @@ pipeline {
             description: 'Test manifest under the manifests folder, e.g. 2.0.0/opensearch-dashboards-2.0.0-test.yml.',
             trim: true
         )
+        string(
+            name: 'INTEG_TEST_JOB_NAME',
+            description: "Name of integration test job that will be triggered, e.g. Playground/integ-test-opensearch-dashboards.",
+            defaultValue: "integ-test-opensearch-dashboards",
+            trim: true
+        )
+        booleanParam(
+            name: 'BUILD_DOCKER',
+            description: 'Build docker image or not.',
+            defaultValue: true
+        )
+        booleanParam(
+            name: 'PUBLISH_NOTIFICATION',
+            description: 'Publish the status of this build job or not.',
+            defaultValue: true
+        )
     }
     stages {
+        stage('verify-parameters') {
+            steps {
+                script {
+                    env.INTEG_TEST_JOB_NAME = INTEG_TEST_JOB_NAME != '' ?
```

Comment:
Do we need the check here since `INTEG_TEST_JOB_NAME` always has the default value which is the same as the `DEFAULT_INTEG_TEST_JOB_NAME` if not assigned?

Reply:
This is probably me being too protective but technically someone can kick off a build and empty out the string for `INTEG_TEST_JOB_NAME`. I can remove this if we do not want to handle those situations.

Reply:
Had some offline discussion with @kavilla, this makes sense to me that we could leave the parameter empty with this stage of check. These situation could happen for example when we pass the params from cron jobs. This shouldn't be a blocker for me.